src/app/core/component.ts

```diff
@@ -200,8 +201,8 @@ export default class Component {
             name: this._name,
             entity: this._entity,
             params: this._params.toJSON(),
-            "x-span": this._xspan,
-            "y-span": this._yspan,
+            "x-span": bounds.width,
+            "y-span": bounds.height,
```

Comment:
An you verify whether `_xspan` and `_yspan` are not retrieving the right values. If thats the case please create getters for those two properties to return the width and height of the bounds